tools/gen-template-doc/gen_template_docs.py

```diff
@@ -153,7 +155,7 @@ def createTemplate(data, path):
                 tdata['objects'][0]['secrets'] = [{"secretNames": secretName}]
 
         # currently only the rhpam-authoring-ha will have clustering section, any new template that supports clustering needs to be added in the clusteringTemplates var.
-        clusteringTemplates = ['rhpam71-authoring-ha.yaml', 'rhpam72-authoring-ha.yaml']
+        clusteringTemplates = ['rhpam71-authoring-ha.yaml', 'rhpam72-authoring-ha.yaml', 'rhpam73-authoring-ha.yaml']
```

Comment:
Just one addition, I think we could add rhdm 72 and 73 here as it supports HA too.

Reply:
@spolti Actually, rhpam authoring ha is in 70, 71, 72, and 73, and rhdm authoring is in 71, 72, and 73 (no 70). So, I've updated accordingly.

Reply:
Awesome, thanks :)